federation/docker-compose-network-ipv4-only.yml

```diff
@@ -19,6 +19,7 @@ services:
       - "conference.xmpp1.localhost.example:172.50.0.10"
       - "xmpp2.localhost.example:172.50.0.20"
       - "conference.xmpp2.localhost.example:172.50.0.20"
+      - "ocsp.localhost.example:172.50.0.30"
```

Comment:
I've switched the IPv4 IP.

I looked at what it would mean to add IPv6 support to the OCSP responder, I think it would add more files and complexity for (I think) little gain. It's not much effort though, so if you think it's worth it I can implement.

Instead, my recent update keeps the OCSP responder on IPv4 only, but makes sure it's properly configured in the hosts file even when using the -6 flag. The OCSP responder will always use IPv4, other services will be able to reach it via IPv4 regardless of whether they're using IPv6 for their own communication.

Reply:
Having OCSP support when running in dualstack mode was the most important bit, thanks for adding that.

There is an ongoing support to get Openfire fully functional with IPv6, so having an OCSP interaction over IPv6 would be a nice-to-have. I totally understand if it's out of scope for what we're doing today though. If it takes a few minutes, please have a look. If it takes significant effort, please leave it as-is.

Reply:
I have added IPv6 support to the OCSP responder. I notice that Dozzle does not work when using IPv6.

Reply:
Ah, to bad. I don't think we'd need that immediately? I've raised a ticket to keep this on the radar: https://github.com/surevine/openfire-docker-compose/issues/71